source/software/matlab_parallel_computing.rst

```diff
@@ -7,117 +7,93 @@ General
+We have Matlab modules available for wICE (in the 2021a toolchain) and for Genius (in the 2021a and 2018a toolchain).
 
-Next, you'll need to load your preferred module, and start it. It is recommended to do this via an interactive job.
+Now, start Matlab and use the function  `configCluster` to set up the cluster profile. If you do not do this, the cluster profile will default to the 'local
+cluster', basically meaning that it will only detect the cores on the node you are currently on.
 
 ::
 
-   $ # Genius
-   $ module use /apps/leuven/skylake/2021a/modules/all
-   $ module load MATLAB/R2022b #MATLAB/R2022a if you would like to load this version
-    
-Once the module is loaded, start Matlab. You can start it in a terminal without the GUI by using the ``-nodisplay`` flag. Then, run the ``configCluster`` script.
-This will set up the parallel configuration.  
+    $ matlab -nodisplay          # this will avoid launching the GUI, but you can do this if you prefer (slow)
+    $ # Within Matlab
+    $ >> configCluster;
 
-::
-
-   $ matlab -nodisplay
+There is a range of additional properties that you can set on the cluster profile. Before you can edit these, you first need to get a handle on the cluster
+profile:
 
 ::
 
-   >> rehash toolbox;
-   >> configCluster;
-   
-You will get a message as follows if it succeeded: 'Complete.  Default cluster profile set to "<cluster_name> <matlab_version>".'. You have now created a cluster
-profile.
+    $ c=parcluster;
 
-Configuring additional properties
-*********************************
-
-It is possible to set some user-specific properties and save them in your cluster profile. You can always adapt these and save the profile again, without performing
-the previous step again. All of these properties are available in the ``AdditionalProperties`` property of your cluster profile. 
-   
-Before you will be able to adapt these properties, you first need to get a handle on the cluster:
+You can then view and edit the additional properties:
 
 ::
 
-   >> c = parcluster;
-   
-``c = parcluster`` will automatically load the correct profile based on the Matlab module you have loaded. So if you would have a profile for both R2022a and R2022b,
-the correct profile will be chosen when using this command.
+    $ # Access
+    $ c.AdditionalProperties
+    $
+    $ #edit
+    $ c.AdditionalProperties.<property_name> = '<property_value>'
+
+One of these properties, you will always have to set, namely `AccountName`. Set this to the name of your credit account.
 
-If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+Next to this, it is also possible to change the more standard parameters of the cluster profile. One of these is the `JobStorageLocation` parameter. This defaults
+to a location in your `$VSC_HOME`. If you are producing a lot of output, it might be good to change this location to e.g. `$VSC_DATA`. Changing this, or any of
```

Comment:
normally users shall avoid using their home folder in all cases. So, please reformulate like:
"It is strongly recommended to change the default storage location to ..."